Tommy-Spurlock/Remember_Me
# Modals with functioning CRUD

Completed Modals and CRUD functionality. 

To test:
-log in and click the add reminder affordance. 
-add some new birthday remindesr. 
-edit reminders
-delete reminders. 
-list should populate with correct information after each new post/edit/delete
